unit-tests/syncer/sw.py

```diff
@@ -116,31 +185,50 @@ def expect( depth_frame = None, color_frame = None, nothing_else = False ):
     Looks at the syncer queue and gets the next frame from it if available, checking its contents
     against the expected frame numbers.
     """
-    global syncer
-    fs = syncer.poll_for_frames()  # NOTE: will never be None
-    if not fs:
+    global syncer, playback_status
+    f = syncer.poll_for_frame()
+    if playback_status is not None:
+        countdown = 50  # 5 seconds
```

Comment:
Why not using a timer class.
Set it to 5 seconds and check has_expired()
It will be more elegant IMO

Reply:
It's easier for me to implement simple code like this without remembering APIs :)